### ISSUE TITLE
Tag account creation source for metrics

### ISSUE THREAD

**Issue by bmann** [fission-suite/fission#365]
To understand how new user accounts are being created, I'd like to "tag" the source of accounts being created, and store that in the database.

The two types of tags to start would be "CLI" for those being created on the command line, and "site.example.com" web location.

So right now web accounts would be coming from "drive.fission.codes", but also could be from "developerapp.fission.app".

Making those analytics available to app owners would also be valuable in the future.

**Comment by expede** [comment]
Totally makes sense. `creationSource` or similar 💯

**Comment by expede** [comment]
🤔 How granular do we want these stats? We can use things like `User-Agent` to track a bunch of stuff like Browser(mobile|desktop)/CLI/Server

**Comment by bmann** [comment]
This shouldn't duplicate running client side analytics.

It just means that it's all in the DB so we can use Metabase to look at this, to answer the question "where / how did this Fission account get created".

**Comment by expede** [comment]
Yup! Just trying to tune which things we care about. Sounds like CLI vs any browser 👍 Do we care about the less likely case where someone is doing this directly via the API on a server?

**Comment by bmann** [comment]
I have no idea how many people would be programmatically creating accounts. If you've got the hood up anyway and want to tag it with API sure, but if it's tough to tag at that level skipping it is fine too.

**Comment by expede** [comment]
> I have no idea how many people would be programmatically creating accounts.

Just thinking about the Python code that was hitting our API for 48-hours at a time. There’s also the Heroku delegating to a FE use case. Depending on how we do it, it could be pretty straightforward to track.

...but also sounds like the main case to check the browser vs CLI 👍
